.ruby-version

```diff
@@ -0,0 +1 @@
+2.1.9
```

Comment:
@kendall-moore: this isn't a version bump―the [`.ruby-version`](https://rvm.io/workflow/projects#project-file-ruby-version) file is used by RVM (and rbenv) to switch to that version of ruby (this case, matching our targets' AIO).

It's probably a good idea to maintain the file.  But if we go this route, we should do it deliberately  it across all the repositories (and keep the corresponding `PUPPET_VERSION` matrix in the `.travis.yml` up-to-date).